### PR TITLE
Added Scalegraser weapons and Cunning weapon fusion

### PR DESCRIPTION
**Pull request by LucD20**
Added scalegraser weapons and the cunning weapon fusion.

### CODE REVIEW

File: src/items/equipment/cunning.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    },
+    "damage": {
+      "parts": []
+    },
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p><span>When you strike a creature with a cunning weapon and deal damage, you can attempt an [[/r 1d20+@skills.eng.mod]]{Engineering}, [[/r 1d20+@skills.lsc.mod]]{Life Science}, or [[/r 1d20+@skills.mys.mod]]{Mysticism} check to identify the creature as appropriate (this takes no action). You gain a +2 circumstance bonus to this check. If you succeed, in addition to learning useful information about the creature, you gain a +1 circumstance bonus to the next saving throw you must attempt against that creature’s abilities within the next minute.</span></p>"
```
Comment (LebombJames) [inline src/items/equipment/cunning.json:48]: Use `@Check` instead. E.g `@Check[type:life-science]`

Also this html is wrapped in a span tag, delete that

File: src/items/equipment/cunning.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        "container": {
+          "actorId": "h0BLCH2Shfu27dYt",
+          "itemId": "POqcKTJa5kYavBC1"
+        },
+        "effectType": "skill",
+        "enabled": false,
+        "max": 2,
+        "modifier": "2",
+        "modifierType": "formula",
+        "notes": "When you strike a creature with a cunning weapon and deal damage, you can attempt an Engineering, Life Science, or Mysticism check to identify the creature as appropriate (this takes no action).",
```
Comment (LebombJames) [inline src/items/equipment/cunning.json:88]: Include the "you gain a +2 to this check" part in each of these modifiers

Comment (LebombJames) [inline src/items/equipment/cunning.json:88]: Include the "you gain a +2 to this check" part in each of these modifiers

**Review by LebombJames — CHANGES_REQUESTED** [review]

**Comment by LucD20** [comment]
Alright, I added the changes.

**Pull request merged**